### PR TITLE
pipeline refresh button and change border of error report and run status

### PR DESCRIPTION
**Pull request by Vishal-raj-1**

### CODE REVIEW

**Review by naveenpaul — CHANGES_REQUESTED** [review]
In label - "Processing..." the three dots should be icon class - icon-tabler-dots from https://tablericons.com/ library. And we should get the pulsating animation. Keep the dots size the same as font size.

File: webapp/styles/main.scss [hunk 1]
```diff
@@ -305,6 +305,65 @@ form {
   tbody td:hover {
     cursor: pointer;
   }
+
+  .processing{
+    background-color: #fff8e6;
```
Comment (mkstayalive) [inline webapp/styles/main.scss]: use scss variables for color codes. All color definitions should go in light.scss.

File: webapp/styles/main.scss [hunk 2]
```diff
@@ -305,6 +305,65 @@ form {
...
+  }
+  
+  .dot-flashing::after {
+    left: 6px;
+    animation-delay: 1s;
+  }
+  
+  @keyframes dotFlashing {
+    0% {
+      background-color: #bb9522;
```
Comment (mkstayalive) [inline webapp/styles/main.scss]: use scss variables for color codes. All color definitions should go in light.scss.

File: webapp/styles/main.scss (hunk 1 above)
Comment (Vishal-raj-1) [reply webapp/styles/main.scss]: it's done

File: webapp/styles/main.scss (hunk 2 above)
Comment (Vishal-raj-1) [reply webapp/styles/main.scss]: Done

**Comment by Vishal-raj-1** [comment]
It is solved

**Pull request closed without merging**